vuepress/docs/next/docs/consume/high-avail-application.md

```diff
@@ -1,25 +1,13 @@
-# Caching and Clustering in an Entando Application
+# High Availability in an Entando Application
 
 ## App Engine Clustering and High Availability
 
 The Entando App Engine can be deployed as a clustered set of instances using the clustering and replication ability of Kubernetes. It is backed by a shared cache with two default choices for implementation.
 
-This guide demonstrates how to configure and test a clustered instance of the Entando App Engine.
+This guide describes how to configure and test a clustered instance of the Entando App Engine.
```

Comment:
need to update this statement, as this page just describes certain aspects about clustered instances